#! /bin/sh -e
## 10_wrapper_uid.dpatch by Tollef Fog Heen <tfheen@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Makes sure we're called with the right UID and GID

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch ${2:+-d $2}}"

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argum
ent"
                exit 1;;
esac

exit 0
@DPATCH@

--- mailman-2.1.4.orig/README.EXIM
+++ mailman-2.1.4/README.EXIM
@@ -72,13 +72,13 @@
 
   # Home dir for your Mailman installation -- aka Mailman's prefix
   # directory.
-  MAILMAN_HOME=/usr/local/mailman
+  MAILMAN_HOME=/var/lib/mailman
   MAILMAN_WRAP=MAILMAN_HOME/mail/mailman
 
   # User and group for Mailman, should match your --with-mail-gid
   # switch to Mailman's configure script.
-  MAILMAN_USER=mailman
-  MAILMAN_GROUP=mailman
+  MAILMAN_USER=list
+  MAILMAN_GROUP=daemon
 
 
 Transport for Exim 3
